.github/workflows/test.yml

```diff
@@ -52,9 +52,10 @@ jobs:
           npm install
 
       - name: Setup PHP
-        uses: nanasess/setup-php@master
+        uses: shivammathur/setup-php@v2
```

Comment:
I've updated the base PHP setup action, to one that's more up to date, and that's already compatible with Ubuntu 22.04.

Reply:
I understand that this action is used to install php 7.4 because it's deprecated in the ubuntu 22.04 image that github provides, right?

Reply:
Actually, this is only a change due to the action not supporting Ubuntu 22.04 yet (I'm sure it'll support it in a future, but it looks like they have a slower pace than in this action).

But in essence, yes, this action is to install PHP 7.4.

Reply:
Okay!

I asked it because i saw this issue where they say that the PHP version for the ubuntu-22.04 is only the 8.1.

https://github.com/actions/virtual-environments/issues/5490

After looking for it i found this https://github.com/marketplace/actions/setup-php-action where they recommend the same action that you use.

Thanks for the answer!